chipseq_workflows/repository_dependencies.xml

```diff
@@ -4,4 +4,5 @@
   <repository toolshed="toolshed.g2.bx.psu.edu" name="bowtie_wrappers" owner="devteam" changeset_revision="df86f29bedee" />
   <repository toolshed="toolshed.g2.bx.psu.edu" name="sam_to_bam" owner="devteam" changeset_revision="d04d9f1c6791" />
   <repository toolshed="toolshed.g2.bx.psu.edu" name="macs2" owner="iuc" changeset_revision="6d4babad010f" />
+  <repository toolshed="testtoolshed.g2.bx.psu.edu" name="split_paired_collection" owner="scottx611x" changeset_revision="429cae20b148" />
```

Comment:
This tool should be published in https://toolshed.g2.bx.psu.edu/view/refinery-platform and the source code should be in https://github.com/refinery-platform/galaxy-tools.